sympy/physics/mechanics/functions.py

```diff
@@ -544,7 +544,7 @@ def find_dynamicsymbols(expression, exclude=None, reference_frame=None):
             i.free_symbols == t_set]) - exclude_set
 
 
-def msubs(expr, *sub_dicts, **kwargs):
+def msubs(expr, *sub_dicts, smart=False, **kwargs):
```

Comment:
Is `**kwargs` still needed?

Reply:
Removing it will break users passing arguments that they expect to work but don't. In this case, I think that's a pretty fair change to make, so will do it.